src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println(this.getClass().getSimpleName() + " starts working.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println(this.getClass().getSimpleName() + " stops working.");
+    }
+}
+

```

Comment:
redundant empty line, we should have only 1 at the end of the file
Hint: check it in the IDE since github displays it a bit differently 😉